packages/config/src/globals.ts

```diff
@@ -108,3 +108,7 @@ export const serverPath = (
 export const databasePath = `${serverPath}db/`
 export const iconUploaderPath = `${serverPath}saveIcon`
 export const hiddenPrefix = '_#_'
+export const SERVER_PING_STATUS = {
+  SUCCEED: 'OK',
```

Comment:
Hello @philb89 - it looks like we don't use `SERVER_PING_STATUS.OK`.  We should, shouldn't we?

Reply:
yep, but I added for the future, may we need to use it :D